pacman/model/constraints/partitioner_constraints/same_atoms_as_vertex_constraint.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, vertex):
         :param ApplicationVertex vertex:
             The vertex to which the constraint refers
         """
+        raise NotImplementedError(
+            "SameAtomsAsVertexConstraint is no longer supported. "
+            "Please contact spinnakerusers@googlegroups.com to discuss your "
+            "requirements.")
         self._vertex = vertex
```

Comment:
unreachable code

Reply:
I sure hope it is unreachable from OUR code otherwise we would have a problem!

As we are unsure if any user was using this constraint this will now fail early and ask him to let us know he was using the this we hope dead code.